decidim-core/app/commands/decidim/unsubscribe_notifications_settings.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This command unsubscribe user from newsletter.
```

Comment:
`s/unsubscribe/unsubscribes/`, maybe?

Reply:
@mrcasals sorry I don't understand you. What do you mean? Change the filename?

Reply:
No, I mean to fix the sentence in the docs to make it gramatically correct (note the verb form)

(that's me being very very very picky 😅 )

Reply:
👌